lib/jellyfish/room.ex

```diff
@@ -458,6 +465,12 @@ defmodule Jellyfish.Room do
     {:noreply, state}
   end
 
+  @impl true
+  def handle_info(%EndpointRemoved{} = track_info, state) do
+    Logger.info("Endpoint removed #{track_info.endpoint_id} removed track #{inspect(track_info)}")
```

Comment:
I don't see any track info in the EndpointRemoved struct -- was this what you meant to do?

```suggestion
  @impl true
  def handle_info(%TrackRemoved{} = track_info, state) do
    Logger.info("Endpoint #{track_info.endpoint_id} removed track #{inspect(track_info)}")
```

Reply:
Almost I was simply lazy. I duplicated lines above and forgot to change them 😅 .